aztec/src/main/kotlin/org/wordpress/aztec/formatting/BlockFormatter.kt

```diff
@@ -209,7 +238,7 @@ class BlockFormatter(editor: AztecText, val listStyle: ListStyle, val quoteStyle
             val endModifier = if (hasLingeringEmptyListItem) 1 else 0
 
             val spans = editableText.getSpans(start, end + endModifier, spanType)
-            spans.forEach { span ->
+            spans.forEach innerForEach@ { span ->
```

Comment:
Niptick - Is `innerForEach@` needed here?

Reply:
Thanks, it's a leftover from a change that required it but eventually got removed.